conda-recipes/llvmdev/cfg_test.ll

```diff
@@ -0,0 +1,56 @@
+; ModuleID = 'foo'
+source_filename = "<string>"
+target datalayout = "e-m:o-i64:64-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-apple-darwin17.4.0"
```

Comment:
Does it matter that the triple is wrong for linux but the test only guards against windows https://github.com/numba/llvmlite/pull/343/files#diff-a5674b1fbcdc1e5882531629a69766cbR71?

Reply:
It doesn't matter as we don't go thru codegen.  We only ask `opt` to make a graph of the CFG.  I have run this on Linux.  In fact, this patch runs fine on our buildfarm.

Reply:
Ok, thanks.